File: src/main/java/com/uwetrottmann/trakt/v2/entities/BaseShow.java
package com.uwetrottmann.trakt.v2.entities;

import org.joda.time.DateTime;

import java.util.List;

public class BaseShow {

    public Show show;

    /** collection, watched */
    public List<BaseSeason> seasons;

    /** collection */
    public DateTime last_collected_at;
    /** watchlist */
    public DateTime listed_at;
    /** watched */
    public Integer plays;
    public DateTime last_watched_at;
    /** progress */
    public Integer aired;
    /** progress */
    public Integer completed;
    /** progress */
    public Episode next_episode;

}

File: src/main/kotlin/com/redissi/trakt/entities/BaseShow.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import java.time.OffsetDateTime

class BaseShow(
    val show: Show? = null,
    /** collection, watched  */
    val seasons: List<BaseSeason>? = null,
    /** collection  */
    @Json(name = "last_collected_at")
    val lastCollectedAt: OffsetDateTime? = null,
    /** watchlist  */
    @Json(name = "listed_at")
    val listedAt: OffsetDateTime? = null,
    /** watched  */
    val plays: Int? = null,
    @Json(name = "last_watched_at")
    val lastWatchedAt: OffsetDateTime? = null,
    @Json(name = "last_updated_at")
    val lastUpdatedAt: OffsetDateTime? = null,
    /** progress  */
    val aired: Int? = null,
    val completed: Int? = null,
    @Json(name = "hidden_seasons")
    val hiddenSeasons: List<Season>? = null,
    @Json(name = "next_episode")
    val nextEpisode: Episode? = null
)